controllers/apply/form-router-next/lib/file-uploads.js

```diff
@@ -109,28 +126,50 @@ function uploadFile({ formId, applicationId, fileMetadata }) {
     });
 }
 
+function checkAntiVirusClamD(formId, applicationId, fileMetadata) {
+    const keyName = [formId, applicationId, fileMetadata.fileData.name].join('/');
+
+    return ClamScan.then(async clamscan => {
+        const {
+            is_infected,
+            viruses
+        } = await clamscan.scan_file(fileMetadata.fileData.path);
+
+        if (is_infected) {
+            logger.error(`${keyName} is infected with ${viruses}`);
+            throw new Error('ERR_FILE_SCAN_INFECTED');
+        } else {
+            return { Key: 'av-status', Value: 'CLEAN' };
+        }
+    }).catch(err => {
+        logger.error(`Attempted read of unscanned file at ${keyName} - ${err.message}`);
```

Comment:
We can write this as `logger.error(`Attempted read of unscanned file at ${keyName}`, err)`. `logger.error` takes an `Error` object as the second argument